pypeit/core/flux_calib.py

```diff
@@ -813,6 +842,9 @@ def get_sensfunc_factor(wave, wave_zp, zeropoint, exptime, tellmodel=None, extin
     else:
         senstot = sensfunc_obs.copy()
 
+    if np.any(sens_fwhm) and np.any(dat_fwhm):
```

Comment:
Just to make sure I understand. `sens_fwhm` and `dat_fwhm` are computed at certain point, passed to this function, but not really used (except for printing their values here). Is that correct? If it is, you should at least mention in the docs.

Reply:
I originally did mean these to be usable to estimate the slit losses, but found that when I implemented the correction it did not help the inconsistency in flux levels between NIRSPEC exposures. So I rolled back the correction step. Though I think it could be valuable to note for the user the potentially different FWHM values for the standard and the data, it is not necessary to have them as new variables.

Reply:
Ok, I will take this out.